### ISSUE TITLE
Thumbnail gallery is not rendering properly

### ISSUE THREAD

**Issue by symeneses** [pymc-devs/pymc-examples#409]
## Issue description
Building the dosc with the current versions (sphinx-gallery-0.11.0, ) is breaking the thumbnail gallery: https://pymcio--404.org.readthedocs.build/projects/examples/en/404/gallery.html.

[This build](https://readthedocs.org/projects/pymc-examples/builds/17519027/) seems to be the last working [build](https://readthedocs.org/projects/pymc-examples/builds/). 

Last working versions:
- sphinx==5.0.2
- sphinx-gallery==0.10.1 

Before this, there was an issue building the docs that was fixed by pinning `ablog<0.10.27` see https://github.com/sunpy/ablog/issues/151. See #404.

**Comment by twiecki** [comment]
Thanks for looking into this @symeneses, want to do a PR to fix it?

**Comment by symeneses** [comment]
> Thanks for looking into this @symeneses, want to do a PR to fix it?

Yes, I will see if there is an error that we have or an issue in the updates. For now, the versions where the gallery is being displayed correctly were pinned in #404.
